angular-client/src/app/app-routing.module.ts

```diff
@@ -14,7 +17,11 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [
+    RouterModule.forRoot(routes),
+    CommonModule,
+    BrowserAnimationsModule // required animations module
```

Comment:
shouldn't need these changes anymore, since you're using Peyton's messageService, pls undo them